devices/fields/spi/spi_v3.yaml

```diff
@@ -0,0 +1,21 @@
+_include:
+  - spi_v3_base.yaml
+  - spi_v3_i2s.yaml
+
+CFG1:
+  UDRDET:
+    StartOfFrame: [0, Underrun is detected at begin of data frame]
+    EndOfFrame: [1, Underrun is detected at end of last data frame]
+    StartOfSlaveSelect: [2, Underrun is detected at begin of active SS signal]
+  UDRCFG:
+    RepeatTransmitted: [2, Slave repeats last transmitted data frame]
```

Comment:
Should there also be the variants for `0` and `1`?

[RM0433](https://www.st.com/resource/en/reference_manual/rm0433-stm32h742-stm32h743753-and-stm32h750-value-line-advanced-armbased-32bit-mcus-stmicroelectronics.pdf) page 2221:
> Bits 10:9 UDRCFG[1:0]: behavior of slave transmitter at underrun condition
>   00: slave sends a constant pattern defined by the user at SPI_UDRDR register
>   01: slave repeats lastly received data frame from master
>   10: slave repeats
>   11: reserved

Reply:
Good catch. Thanks

Reply:
Reverted. Already present in `spi_v3_base.yaml`.